### PR TITLE
Close function is modified, VMs can be shutdown properly.

### PR DESCRIPTION
**Pull request by mrturkmen06**
The issue regarding to VMs (which do not shutdown properly) fixed, so whenever an event stopped, VMs will be destroyed which provides more resources than before.

### CODE REVIEW

File: lab/lab.go [hunk 1]
```diff
@@ -208,24 +207,26 @@ func (l *lab) Restart(ctx context.Context) error {
...
+	 	wg.Add(1)
+	 	go func (){
+	 		// closing VMs....
+	 		defer wg.Done()
+			if err := lab.vm.Close(); err!=nil{
+				log.Error().Msgf("Error on Close function in lab.go %s",err)
+			}
+		}()
+	 }
+	 go func() {
```
Comment (kdhageman) [inline lab/lab.go]: no `wg.Add(1)`?

File: lab/lab.go [hunk 2]
```diff
@@ -208,24 +207,26 @@ func (l *lab) Restart(ctx context.Context) error {
...
-	}
-
-	if l.environment != nil {
-		wg.Add(1)
-		go closer(l.environment)
-	}
-
+	 for _, lab  := range l.frontends {
+	 	wg.Add(1)
+	 	go func (){
```
Comment (kdhageman) [inline lab/lab.go]: I would suggest to pass the lab as a parameter to the goroutine function something like this:
```
go func (lab lab.Lab){
	// do things with lab
}(lab)
```

Comment (mrturkmen06) [reply lab/lab.go]: What is the main reason of that ?

File: lab/lab.go (hunk 1 above)
Comment (mrturkmen06) [reply lab/lab.go]: ups, added.

File: lab/lab.go (hunk 2 above)
Comment (mrturkmen06) [reply lab/lab.go]: discussed.

**Pull request merged**